takari-builder-enforcer/src/test/java/io/takari/builder/enforcer/modularity/MavenProjectContextTest.java

```diff
@@ -90,17 +92,29 @@ public void testLoad() throws Exception {
 
     assertProjextContext(ctx, //
         "+E p4" //
-        , "+R /locations/dirAllow/" //
-        , "+W /locations/fileAllow" //
-        , "-R /locations/dirForbid/" //
-        , "-W /locations/fileForbid" //
-        , "/locations/" //
+        , rule("+R ", "/locations/dirAllow/") //
```

Comment:
what do you gain from this method?

Reply:
with this method you can kinda see what is expected, but the expected contents becomes almost unreadable if I inline `normalize0` here.